addons/armaos/functions/fnc_terminal_init.sqf

```diff
@@ -72,8 +72,20 @@ private _localGameLanguage = language;
 // if the language is german, it's obvious, that the keyboard layout is also german (this is not the case, if game language is english)
 // perhaps we need to provide a CBA setting for changing keyboard layout or allow to change the layout directly in the terminal window
 
+private _terminalKeyboardLayout = "";
+
 //AE3_KeyboardLayout is a CBA setting
-private _terminalKeyboardLayout = AE3_KeyboardLayout;
+if (!isMultiplayer || (isServer && hasInterface)) then
```

Comment:
`!isMultiplayer` should be redundand

Reply:
Sorry, that's not the same.

`!isMultiplayer` means Singleplayer 
`isServer && hasInterface` means local hosted Multiplayer

In both cases not the CBA client settings but the CBA server settings are used